### PR TITLE
Anomaly Detection

### PR DESCRIPTION
**Pull request by conorsim**
Anomaly detection experiment using the PaDiM model trained on wood.

### CODE REVIEW

File: gen2-anomaly-detection/download.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import gdown
+
+if __name__ == "__main__":
+
+    # Download blob from Google Drive
+    url = "https://drive.google.com/uc?id=1qZEaadgleeazz-Pe4I-yqdUqKr4qMCmv"
```
Comment (Erol444) [inline gen2-anomaly-detection/download.py:6]: move to depthai-model-zoo and use blobconverter to download the blob

**Review by Erol444 — APPROVED** [review]

**Comment by conorsim** [comment]
@hardikdava just to make sure, you mean the model conversion/export process, right? It's using anomalib, but I could add a notebook or conversion script and README.

**Comment by hardikdava** [comment]
@conorsim yes, I meant how can I export and convert model. A notebook would be great. Thanks

**Comment by conorsim** [comment]
@hardikdava new PR here: [https://github.com/luxonis/depthai-experiments/pull/446](https://github.com/luxonis/depthai-experiments/pull/446)

**Pull request merged**